### PR TITLE
create react component to display design

### PR DESCRIPTION
**Pull request by LeslieAine**
In this PR I;
- Deleted all the boilerplate from Create React App (text, images, styles).
- Created a new directory called components.
- Inside components, created a new Calculator.js file.
- In Calculator.js, created a React component that matches this design:

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+      <button className="calculator-button right" type="button"> x </button>
+      <button className="calculator-button" type="button"> 4 </button>
+      <button className="calculator-button" type="button"> 5 </button>
+      <button className="calculator-button" type="button"> 6 </button>
+      <button className="calculator-button right" type="button"> - </button>
+      <button className="calculator-button" type="button"> 1 </button>
+      <button className="calculator-button" type="button"> 2 </button>
+      <button className="calculator-button" type="button"> 3 </button>
+      <button className="calculator-button right" type="button"> + </button>
+      <button className="calculator-button" id="zero" type="button"> 0 </button>
```
Comment (Munsa1) [inline src/components/Calculator.js:23]: - [x] Kindly style your calculator component a little bit moe so as to match the given design better. Adding a width property to the main container of the calculator and setting it to `30vw` achieves this as well as reducing the `grid-gap` to `1px` improves the design much more.

Before
image


After adding further styling
image


Given design
image

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @LeslieAine, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
Passing Linters :heavy_check_mark: 
Correct GitHub Flow :heavy_check_mark: 
Descriptive ReadMe File :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._
- [x] Kindly remove all boilerplate code created with CRA command
image


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js (hunk 1 above)
Comment (LeslieAine) [reply src/components/Calculator.js:23]: Thank you for the review and suggestions. Really helpful!

**Review by julie-ify — APPROVED** [review]
Hi @LeslieAine,

## Highlights!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Status: Approved ✔️ 

Cheers and Happy coding!👏👏👏

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**